app/views/playlist_items/_markers.html.erb

```diff
@@ -13,7 +13,7 @@ Unless required by applicable law or agreed to in writing, software distributed
   specific language governing permissions and limitations under the License.
 ---  END LICENSE_HEADER BLOCK  ---
 %>
-<div id="markers" class="container">
+<div id="markers" class="container" data-can-edit-playlist-item="<%= can? :edit, @playlist_item %>">
```

Comment:
I'm wondering if we should do this more universally and get rid of all marker abilities since they are tied to the playlist item.  This would probably make #2561 simpler.

Reply:
Makes sense. Maybe we could put this attribute on each item in the right-hand panel?

Reply:
I don't think we need that yet since this seems to work for now.

Reply:
Oh. Then I guess I don't know what you meant.